src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/nebula/tools/generator/v2/SparkClientGenerator.scala

```diff
@@ -310,7 +309,11 @@ object SparkClientGenerator {
               val values = (for {
                 property <- valueProperties if property.trim.length != 0
               } yield extraValue(row, property)).mkString(",")
-              (String.valueOf(extraValue(row, vertex)), values)
+              (if (String.valueOf(extraValue(row, vertex)).startsWith("\"") && String.valueOf(extraValue(row, vertex)).endsWith("\"")) {
+                String.valueOf(extraValue(row, vertex)).slice(1,-1)
```

Comment:
seems `extraValue` will append `quotation marks` at begin and end of the string

Reply:
By the way, you should extraValue only once

Reply:
U're right. Better way to fix the problem is use another function instead of extraValue to especially deal with index rather than
 using extraValue.